src/rust/engine/fs/fs_util/src/main.rs

```diff
@@ -272,6 +272,9 @@ fn execute(top_match: clap::ArgMatches) -> Result<(), ExitError> {
               .map(|s| s.to_string())
               .collect::<Vec<String>>(),
             &[],
+            // By using `Ignore`, we assume all elements of the globs will definitely expand to
```

Comment:
I believe that this case could at least warn if something isn't matched, but @illicitonion could confirm.